elektronLogging/clsfnTaskDistOverheadLogger.go

```diff
@@ -0,0 +1,49 @@
+package elektronLogging
+
+import (
+	"os"
+	log "github.com/sirupsen/logrus"
+)
+
+type ClsfnTaskDistOverheadLogger struct {
+	LoggerImpl
+}
+
+func NewClsfnTaskDistOverheadLogger(logType int, prefix string) *ClsfnTaskDistOverheadLogger {
+	cLog := new(ClsfnTaskDistOverheadLogger)
```

Comment:
Use struct literals instead of new as it's more descriptive:
`&ClsfnTaskDistOverheadLogger{}`

Reply:
Okay, will do the changes